metadata-io/src/main/java/com/linkedin/metadata/graph/neo4j/Neo4jGraphService.java

```diff
@@ -524,4 +747,19 @@ private Statement getOrInsertNode(@Nonnull Urn urn) {
   public boolean supportsMultiHop() {
     return true;
   }
+
+  private boolean getRealDirection(@Nonnull String sourceType, @Nonnull String relationshipType) {
```

Comment:
Would you mind explaining in more detail what this function does? If so, could you update the comment?

I'm also wondering if there is a better name than getRealDirection, especially since this returns a boolean

Reply:
change name to reverseSourceDest and added comments